lib/active_merchant/billing/gateways/nmi.rb

```diff
@@ -166,7 +166,10 @@ def add_payment_method(post, payment_method, options)
         elsif payment_method.is_a?(NetworkTokenizationCreditCard)
           post[:ccnumber] = payment_method.number
           post[:ccexp] = exp_date(payment_method)
-          post[:token_cryptogram] = payment_method.payment_cryptogram
```

Comment:
❓ Question:

with this change you mean that `token_cryptogram` is wrong for general EMVco Tokens ? and all apple pay / Google pay and VISA/MC tokens should use `cavv` ?

I'm asking just because backwards compatibility

Reply:
🤔 🤔 sounds like a CAB process if do that move, what if just in case we leave that `token_cryptogram` field there for Emco tokens and use `cavv` for the ones with apple/google on it's source.

Reply:
I agree, sound more reasonable, I will do the change